platform/viewer/src/components/ViewportGrid/ViewportGrid.js

```diff
@@ -37,7 +37,7 @@ const ViewportGrid = function(props) {
     viewportData.forEach(displaySet => {
       loadAndCacheDerivedDisplaySets(displaySet, studies);
     });
-  }, [viewportData]);
+  }, [studies, viewportData]);
```

Comment:
I think this is fine. I wish it was a bit cleaner as the "watch" is a lot of properties. Probably fine for now, but we should address this in the redesign.